hyp3_testing/helpers.py

```diff
@@ -20,9 +20,9 @@ def sort_jobs_by_parameters(jobs: Batch) -> Batch:
     return Batch(sorted_jobs)
 
 
-def get_jobs_in_environment(job_name, api):
+def get_jobs_in_environment(job_name: str, api: str, user_id: Optional[str]) -> Batch:
```

Comment:
The type annotation says `user_id: Optional[str]` but no default value is provided, we should change it to `str` or provide the default value.

Reply:
It's definitely clearer with a default of `None`; I'll change that. 

Interstingly, `Optional[thing]` is just short for `Union[None, thing]` and specifying the *value*, while setting a default makes the *argument*  optional (you can require an argument that can be `None` or a `str`, but I can't think of a use case where it wouldn't be better set a default of `None`. :thinking: )
https://docs.python.org/3/library/typing.html#typing.Optional